compiler/rustc_codegen_llvm/src/consts.rs

```diff
@@ -196,16 +196,10 @@ fn check_and_apply_linkage<'ll, 'tcx>(
             g2
         }
     } else if cx.tcx.sess.target.arch == "x86"
+        && common::is_mingw_gnu_toolchain(&cx.tcx.sess.target)
```

Comment:
We don't need to use a decorated name for msvc? (Or is the sym already decorated in this case earlier?)

Reply:
Correct, we no longer need the decorated name because the import library has both the decorated and undecorated names.

Reply:
Got it, thanks!